src/utils.ts

```diff
@@ -18,11 +18,13 @@
 import * as vscode from 'vscode';
 import * as extension from './extension';
 import * as fs from 'fs';
-import * as path from 'path';
+import { ViewColumn } from 'vscode';
+
 
 export const DIDACT_DEFAULT_URL : string = 'didact.defaultUrl';
 export const DIDACT_REGISTERED_SETTING : string = 'didact.registered';
 export const DIDACT_NOTIFICATION_SETTING : string = 'didact.disableNotifications';
+export const DIDACT_COLUMN_SETTING : string = 'didact.lastColumnUsed';
```

Comment:
what about naming DIDACt_LAST_COLUMN_SETTING to reflect more closely the behavior